src/Uno.UI/UI/Xaml/Controls/Image/Image.wasm.cs

```diff
@@ -102,7 +106,15 @@ private void OnSourceChanged(DependencyPropertyChangedEventArgs e)
 
 			_lastMeasuredSize = _zeroSize;
 
-			if (source is WriteableBitmap wb)
+			var stream = source?.Stream;
+			if (stream != null)
+			{
+				stream.Seek(0, SeekOrigin.Begin);
+				var encodedBytes = Convert.ToBase64String(stream.ReadBytes());
+				var url = $"data:application/octet-stream;base64,{encodedBytes}";
```

Comment:
No need to use interpolation here:
```suggestion
				var url = "data:application/octet-stream;base64," + encodedBytes;
```